pkg/operator/file/ceph/mds_test.go

```diff
@@ -83,6 +86,40 @@ func TestStartMDS(t *testing.T) {
 	err = CreateFilesystem(context, fs, "v0.1", false, []metav1.OwnerReference{})
 	assert.Nil(t, err)
 	validateStart(t, context, fs)
+
+	// Test multiple filesystem creation
+	executor = &exectest.MockExecutor{
+		MockExecuteCommandWithOutputFile: func(debug bool, actionName string, command string, outFileArg string, args ...string) (string, error) {
+			if contains(args, "ls") {
+				return fses, nil
+			}
+			return "{\"key\":\"mysecurekey\"}", errors.New("multiple fs")
+		},
+		MockExecuteCommandWithOutput: func(debug bool, actionName string, command string, args ...string) (string, error) {
+			if strings.Contains(command, "ceph-authtool") {
```

Comment:
the `command` doesn't equal `ceph-authtool`? we rely on the tools being in the path so that should be fine

Reply:
The [ListFilesystems command](https://github.com/rook/rook/blob/master/pkg/daemon/ceph/client/filesystem.go#L63) calls [CephTool](https://github.com/rook/rook/blob/master/pkg/daemon/ceph/client/command.go#L32) and not ceph-authtool. Don't think ceph-authtool is being called here.

Reply:
ListFilesystems calls `ceph fs ls`. I believe `ceph-authtool` would be called when the new mds instance is configured, but I'm not sure we will hit that code path since we expect to fail the 2nd file system